Volume/Views/UnderlinedText.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  UnderlinedText.swift
+//  Volume
+//
+//  Created by Daniel Vebman on 11/4/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct UnderlinedText: View {
+    
+    @State var text: String
+    @State private var textSize: CGSize = .zero
+    
+    init(_ text: String) {
+        _text = State(initialValue: text)
+    }
+
+    var body: some View {
+        VStack {
+            Text(text)
+                .fixedSize()
+                .background(SizeGetter(size: $textSize))
+
+            Image("underline")
+                .resizable()
+                .scaledToFit()
+                .frame(width: textSize.width + 4, height:
+                    (textSize.width + 4) * 6 / 112)
+                .clipped()
+                .padding(.top, -10 * textSize.height / 25)
+        }
+    }
+    
+}
+
+struct UnderlinedText_Previews: PreviewProvider {
+    static var previews: some View {
+        PreviewWrapper()
+    }
+
+    struct PreviewWrapper: View {
+        var body: some View {
+            UnderlinedText("THE BIG READ AND OTHER THINGS")
+                .font(.custom("Futura-Bold", size: 15))
+        }
+    }
+}
+
+struct SizeGetter: View {
```

Comment:
If you see us using this in any other file, we should probably make it its own file

Reply:
Totally agree. I think I'll leave it like this for now, but if we do use it elsewhere we should move it. The only reason I'm hesitant to move it now is because I want to de-incentivize us from relying on those kinds of tricks.